qml/components/AppInformation.qml

```diff
@@ -0,0 +1,125 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+BackgroundItem {
+    property bool _expanded: false
+
+    property var pkg
+
+    id: infoItem
+    width: parent.width
+    height: content.height
+    clip: true
+    highlighted: down && !highlightDelayTimer.running
+    _showPress: highlighted
+    propagateComposedEvents: true
+
+    onPressedChanged: {
+        if (pressed) {
+            highlightDelayTimer.restart()
+        }
+    }
+
+    onClicked: _expanded = !_expanded
+
+    Behavior on height { NumberAnimation { duration: 200; easing.type: Easing.InOutQuad } }
+
+    Timer {
+        // Idea is from the official store client
+        id: highlightDelayTimer
+        interval: 50
+    }
+
+    Column {
+        id: content
+        anchors {
+            left: parent.left
+            right: parent.right
+            leftMargin: Theme.horizontalPageMargin
+            rightMargin: Theme.horizontalPageMargin
+        }
+
+        Item {
+            width: parent.width
+            height: bodyLabel.height
+
+            Label {
+                id: bodyLabel
+                width: parent.width
+                maximumLineCount: _expanded ? 0 : 7
+                text: pkg.description
+                color: infoItem.highlighted ? Theme.highlightColor : Theme.primaryColor
+                linkColor: Theme.highlightColor
+                font.pixelSize: Theme.fontSizeSmall
```

Comment:
For testing, I tried with normal font size and, personally, it seems to be better. One disadvantage could be that clickable elements, such as "Releases" and "Issues", are of the same size. However, those elements are separated significantly away and have an arrow button next to them to show the difference. In this respect, maybe we should switch to normal fonts and do the size bump for ChumDetail

Reply:
Lets try with normal (no size set) for description, and Small for ChumDetail

Reply:
I'd prefer that as well. We can revert later if users complain about it :).